src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/ExternalUsersApiService.kt

```diff
@@ -67,13 +63,12 @@ class ExternalUsersApiService(
       .bodyToMono(RolesPaged::class.java)
       .block()!!
 
-  @Throws(RoleNotFoundException::class)
   fun getRoleDetail(roleCode: String): Role =
     externalUsersWebClient.get()
       .uri("/roles/$roleCode")
       .retrieve()
       .bodyToMono(Role::class.java)
-      .block(timeout) ?: throw RoleNotFoundException("get", roleCode, "notfound")
+      .block() ?: throw RoleNotFoundException("get", roleCode, "notfound")
```

Comment:
I thought we'd decide just to ignore this if it failed?

Reply:
removed timeout (10s) on block() method which is far more than http timeout set as 1s (hmpps-auth.endpoint.timeout)